decentraland/unity-explorer
# fix: speed up teleport to roads

## What does this PR change?

Adds roads to scene cache to make the loading screen dissapear as it should

## How to test the changes?

1. Launch the explorer
2. Teleport to 150,0 or any road coordinate. Loading screen should not get stuck at 92%
3. Teleport to any lod scene (ie: North or east plaza). First time it should take a little bit of time. Teleport away and back. it should be faster now
5. Close DCL and reopen. Teleport to the lod scene. It should be faster than the first time

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
